src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -149,9 +149,10 @@ public function replace($path, $content)
     /**
      * Prepend to a file.
      *
-     * @param  string  $path
-     * @param  string  $data
+     * @param  string $path
+     * @param  string $data
```

Comment:
You're going to need to revert all these param space changes. There should be 2 spaces before and after the param type.